static/app/views/dashboards/widgetBuilder/components/widgetBuilderSlideout.tsx

```diff
@@ -58,6 +64,11 @@ function WidgetBuilderSlideout({isOpen, onClose, onSave}: WidgetBuilderSlideoutP
         <Section>
           <WidgetBuilderQueryFilterBuilder />
         </Section>
+        {isChartWidget && (
+          <Section>
+            <WidgetBuilderGroupBySelector />
+          </Section>
+        )}
```

Comment:
I haven't implemented any handling of switching over to a different widget type because i'm not sure if that's something we would need to tackle later or not. If anything I can do it in a different PR

Reply:
Yeah we can tackle it later, it's a different set of work :pray: